src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInListener.cs

```diff
@@ -60,11 +63,21 @@ public override void OnStartActivity(Activity activity, object payload)
                 return;
             }
 
+            // Create a brand new activity using ActivitySource
```

Comment:
I added this code into existing place itself, which does span population. Once we settle on approach, we'll be removing all the span populations and replace with activity.